README.md

```diff
@@ -70,8 +70,8 @@ $ kas build meta-iot2000-bsp/kas.yml
 You can also reproduce the Windows or Linux SDK this way:
 
 ```shell
-$ kas build meta-iot2000-example/sdk-kas-windows-i586.yml
-$ kas build meta-iot2000-example/sdk-kas-linux-x86.yml
+$ kas build meta-iot2000-example/kas-sdk-windows-i586.yml
+$ kas build meta-iot2000-example/kas-sdk-linux-x64.yml
```

Comment:
Good catch, but we should split this up into a separate commit.

Reply:
Ok, I created a new pull request for this file only